python/lsst/ctrl/oods/cacheCleaner.py

```diff
@@ -185,14 +187,15 @@ async def getDirectoriesOlderThan(self, seconds, directory):
         """
         directories = []
 
-        for root, dirs, files in os.walk(directory, topdown=False):
-            for name in dirs:
-                await asyncio.sleep(0)
-                full_name = os.path.join(root, name)
-                stat_info = os.stat(full_name)
-                mtime = stat_info.st_mtime
-                if mtime < seconds:
-                    directories.append(full_name)
+        scanner = Scanner()
+        async for entry in scanner.scan(directory):
+            if entry.is_file():
+                continue
+            full_name = entry.path
+            stat_info = os.stat(full_name)
+            mtime = stat_info.st_mtime
```

Comment:
I _know_ it's very nitpicky, but `getFilesOlderThan` uses `modification_time` as a label for the same piece of data. Could we keep the naming  consistent between these methods? (I'll be fine with either version as long as it is used consistently.) :slightly_smiling_face:

Reply:
Renamed.